content/blog/newsletter_2022-02-05.md

```diff
@@ -0,0 +1,232 @@
+---
+title: The XMPP Newsletter December 2021 & January 2022
+author: emus
+date: 2022-02-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter covering the month of December 2021 and January 2022!
+
+We hope you had a great shift into the new year by now as well as are happy to have you reading the new release! We guess that this episode has caught some weight over the new year's holidays :-)
+
+Many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider saying thanks or help these projects!
+
+[Read this Newsletter via our RSS Feed](https://xmpp.org/feeds/all.atom.xml)!
+
+Interested in supporting the Newsletter team? Read more at the bottom.
+
+Other than that — enjoy reading!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here (with some delay):
+
+- The French translation appears at [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- The German translation appears at [anoxinon.de](https://anoxinon.de/blog/)
+- The Italian translation appears at [NicFab.it](https://www.nicfab.it/)
+- The Spanish translation appears at TBA
```

Comment:
Can we already define how the link would look like?

Another question: Could we link with this rendering to the external ones? For someone clicks French and will be redirected?